Specs/Scene/Batched3DModel3DTileContentProviderSpec.js

```diff
@@ -0,0 +1,214 @@
+/*global defineSuite*/
+defineSuite([
+        'Scene/Batched3DModel3DTileContentProvider',
+        'Core/Cartesian3',
+        'Core/Color',
+        'Core/defaultValue',
+        'Core/defined',
+        'Core/HeadingPitchRange',
+        'Core/loadArrayBuffer',
+        'Scene/Cesium3DTileContentState',
+        'Scene/Cesium3DTileset',
+        'Scene/Model',
+        'Specs/createScene',
+        'Specs/pollToPromise',
+        'ThirdParty/when'
+    ], function(
+        Batched3DModel3DTileContentProvider,
+        Cartesian3,
+        Color,
+        defaultValue,
+        defined,
+        HeadingPitchRange,
+        loadArrayBuffer,
+        Cesium3DTileContentState,
+        Cesium3DTileset,
+        Model,
+        createScene,
+        pollToPromise,
+        when) {
+    "use strict";
+
+    var scene;
+    var centerLongitude = -1.31995;
+    var centerLatitude = 0.69871;
+
+    var withBatchTableUrl = './Data/Cesium3DTiles/Batched/BatchedWithBatchTable/';
+    var withoutBatchTableUrl = './Data/Cesium3DTiles/Batched/BatchedWithoutBatchTable/';
+
+    beforeAll(function() {
+        scene = createScene();
+
+        // One building in each data set is always located in the center, so point the camera there
+        var center = Cartesian3.fromRadians(centerLongitude, centerLatitude);
+        scene.camera.lookAt(center, new HeadingPitchRange(0.0, -1.57, 15.0));
+    });
+
+    afterAll(function() {
+        scene.destroyForSpecs();
+    });
+
+    afterEach(function() {
+        scene.primitives.removeAll();
+    });
+
+    function verifyRender(tileset) {
```

Comment:
Are we ready to make a helper class to reuse this function and friends with Instanced3DModel3DTileContentProviderSpec.js and future tests?